statistics/handle/update.go

```diff
@@ -1042,11 +1053,14 @@ func (h *Handle) DumpFeedbackForIndex(q *statistics.QueryFeedback, t *statistics
 		rangeFB := &statistics.QueryFeedback{PhysicalID: q.PhysicalID}
 		// prefer index stats over column stats
 		if idx := t.IndexStartWithColumn(colName); idx != nil && idx.Histogram.Len() != 0 {
-			rangeCount, err = t.GetRowCountByIndexRanges(sc, idx.ID, []*ranger.Range{&rang})
+			rangeCount, err = t.GetRowCountByIndexRanges(sc, idx.ID, []*ranger.Range{rang})
 			rangeFB.Tp, rangeFB.Hist = statistics.IndexType, &idx.Histogram
 		} else if col := t.ColumnByName(colName); col != nil && col.Histogram.Len() != 0 {
-			rangeCount, err = t.GetRowCountByColumnRanges(sc, col.ID, []*ranger.Range{&rang})
-			rangeFB.Tp, rangeFB.Hist = statistics.ColType, &col.Histogram
+			err = convertRangeType(rang, col.Tp, sc.TimeZone)
```

Comment:
should we use `time.UTC`?

Reply:
Yes, in fact the `sc` defined in line 1014 is already `time.UTC`.

Reply:
I think directly pass `time.UTC` can improve the code readability.